src/Modules/OrchardCore.Commerce/Abstractions/IStripePaymentService.cs

```diff
@@ -31,9 +31,15 @@ public interface IStripePaymentService
     Task<ContentItem> CreateOrUpdateOrderFromShoppingCartAsync(PaymentIntent paymentIntent, IUpdateModelAccessor updateModelAccessor);
 
     /// <summary>
-    /// Updates the corresponding order status to Order for the given <paramref name="paymentIntent"/>.
+    /// Creates an order content item without payment in the database based on the current <see cref="ShoppingCart"/> content.
     /// </summary>
-    Task UpdateOrderToOrderedAsync(PaymentIntent paymentIntent);
+    Task<ContentItem> CreateNoPaymentOrderFromShoppingCartAsync(IUpdateModelAccessor updateModelAccessor);
```

Comment:
Why do you pass in the `IUpdateModelAccessor` as an argument? It should be provided to the service via dependency injection. (that's what already happens in the `PaymentController` where you call it)